src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -43,7 +43,8 @@ public void fillResponseTextBox(int qnNumber, int responseNumber, String text) {
         WebElement element = browser.driver.findElement(
                 By.name(Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-" + qnNumber + "-" + responseNumber));
         click(element);
-        fillTextBox(element, text);
+        element.clear();
+        element.sendKeys(text);
```

Comment:
It should be enough. Initially, I changed it to fillRichTextEditor() but such approach doesn't handle symbols counter, so I came back. I'll use fillTextBox(), it seems to be the same